### PR TITLE
Updated Readme to indicate new features and contributors

### PR DESCRIPTION
**Pull request by pjmsullivan**
# Checklist

- [ ] Bugfix
- [ ] New feature
- [X] Refactor

# Related Issue

- Updated readme to indicate new features and contributors

# Solution

- Updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,10 +63,10 @@ import { ObsidianRouter, gql } from 'https://deno.land/x/obsidian/mod.ts';
 
 const PORT = 8000;
 
-const app = new Application();
+const app = new Application(https://github.com/);
```
Comment (cssim22) [inline README.md:66]: What does the https://github.com/ do on all these parameters?

**Review by cssim22 — CHANGES_REQUESTED** [review]
Need to remove the github links

File: README.md (hunk 1 above)
Comment (pjmsullivan) [reply README.md]: Fixed inadvertent changes.

**Review by cssim22 — APPROVED** [review]
All looks good!

**Review by justinwmckay — APPROVED** [review]

**Review by raymondcodes — APPROVED** [review]
Approved Readme

**Pull request merged**